sources/errors.swift

```diff
@@ -1,8 +1,22 @@
+import Foundation
+
+enum CommandLineError: Error, CustomStringConvertible {
+    case invalidUUID(string: String)
+
+    var description: String {
+        switch self {
+        case .invalidUUID(let string):
+            return "The UUID '\(string)' is invalid"
+        }
+    }
+}
+
 enum SimulatorFetchError: Error, CustomStringConvertible {
     case simctlFailed
     case failedToReadOutput
     case noBootedSimulators
     case noMatchingSimulators(name: String)
+    case noMatchingUDID(udid: UUID)
```

Comment:
Not sure if intentional, but might want to double check UDID vs. UUID usage

Reply:
It is intentional, but it's still confusing. I would use `udid` everywhere but the `UUID` type makes that difficult. But from the simctl json it's `udid`